### PR TITLE
add zh-tw README

### PR DESCRIPTION
**Pull request by tigercosmos**
Add the traditional Chinese(zh-tw) version of README.

The preview can be checked here: https://github.com/tigercosmos/PcapPlusPlus/blob/zh-tw-doc/translation/README-zh-tw.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +29,7 @@ PcapPlusPlus enables decoding and forging capabilities for a large variety of ne
   - [Vcpkg](#vcpkg)
   - [Conan](#conan)
   - [Build It Yourself](#build-it-yourself)
+  - [Verify your packages](#verify-your-packages)
```
Comment (tigercosmos) [inline README.md:32]: This is missed from the previous PR.

File: README.md [hunk 2]
```diff
@@ -18,6 +18,8 @@
 
 PcapPlusPlus enables decoding and forging capabilities for a large variety of network protocols. It also provides easy to use C++ wrappers for the most popular packet processing engines such as [libpcap](https://www.tcpdump.org/), [WinPcap](https://www.winpcap.org/), [Npcap](https://nmap.org/npcap/), [DPDK](https://www.dpdk.org/), [eBPF AF_XDP](https://www.kernel.org/doc/html/next/networking/af_xdp.html) and [PF_RING](https://www.ntop.org/products/packet-capture/pf_ring/).
 
+Translations: [正體中文](./translation/README-zh-tw.md)
```
Comment (tigercosmos) [inline README.md:21]: All translations will be here.

File: translation/README-zh-tw.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+[![Codecov](https://img.shields.io/codecov/c/github/seladb/PcapPlusPlus?logo=codecov&logoColor=white)](https://app.codecov.io/github/seladb/PcapPlusPlus)
+[![OpenSSF Scorecard](https://api.scorecard.dev/projects/github.com/seladb/PcapPlusPlus/badge)](https://scorecard.dev/viewer/?uri=github.com/seladb/PcapPlusPlus)
+[![GitHub 貢獻者](https://img.shields.io/github/contributors/seladb/PcapPlusPlus?style=flat&label=Contributors&logo=github)](https://github.com/seladb/PcapPlusPlus/graphs/contributors)
+
+[![X 關注](https://img.shields.io/badge/follow-%40seladb-1DA1F2?logo=x&style=social)](https://x.com/intent/follow?screen_name=seladb)
+[![GitHub Repo 星星](https://img.shields.io/github/stars/seladb/PcapPlusPlus?style=social)]()
+
+</div>
+
+[PcapPlusPlus](https://pcapplusplus.github.io/) 是一個跨平台的 C++ 函式庫，提供高效、強大且易於使用的功能，來進行網路封包的捕捉、解析和生成。
```
Comment (tigercosmos) [inline translation/README-zh-tw.md:16]: I used ChatGPT to translate first, and fine-tuned the translation.

**Review by seladb — COMMENTED** [review]
@tigercosmos I wonder how we plan to maintain this translation, since only you know Chinese 🤔 

Also - is there a way to fail CI if `README.md` is updated and this file isn't?

**Comment by tigercosmos** [comment]
@seladb As we don't have many people on the team now, the translation can be reviewed by the public later if there are any issues. Same for all other languages.

For the CI, I think I can create a script, if there is any modification on `README`, we detect if the files under `translation` are also updated. But I guess we can put this topic to the next PR?

**Comment by tigercosmos** [comment]
BTW, our previous contributor @zhengfeihe (Chinese) speaks Mandarin as well, maybe he could take a look. @zhengfeihe  would you like to help review this translation?

File: README.md (hunk 2 above)
Comment (clementperon) [reply README.md:21]: What about this pattern ? https://github.com/jonatasemidio/multilanguage-readme-pattern/tree/master

Comment (tigercosmos) [reply README.md:21]: @clementperon  My concern is that there are many documents, e.g. CONTRIBUTING.md (, which is at the root), and we will also have translation files for them. If we follow this pattern, the root directory will have a lot of translation files.

**Comment by tigercosmos** [comment]
@seladb Maybe we can merge the PR? Or do you prefer to find some native speakers to take a look?

**Comment by seladb** [comment]
> @seladb Maybe we can merge the PR? Or do you prefer to find some native speakers to take a look?

It'd be great if another native speaker could review. @tigercosmos do you know someone who can help?

**Comment by tigercosmos** [comment]
@zhengfeihe  can you help?

**Comment by tigercosmos** [comment]
Let me invite @csinrn, my Taiwanese friend, to help with this. @csinrn Could you help review it?

**Comment by csinrn** [comment]
Thanks for the invitation. Glad we have a Chinese version README for a good library to use. Below is some comments from me:

* The topic "自行建構", I would prefer use "自行建置" to align with the "從原始碼建置" later.
* "封包捕獲" is ok and understandable. To make it more colloquial, I think we can change to either "捕捉"  (a mainland usage, [reference](https://baike.baidu.com/item/%E6%8A%93%E5%8C%85/9929103)), or  "擷取" (tw usage, [reference](https://learn.microsoft.com/zh-tw/azure/network-watcher/packet-capture-overview)).
* "封包解析與構造": Same here. We can use "解析與建構" to align with the "網路封包建構" later in the sentence.
* "線速封包處理": To me, we can remove 線速 and explain it later. Like "封包處理: 提供線性速度且易用的...".
* "封包尾部: Let's not translating this. Keep it "Packet trailer" would be better.

**Comment by tigercosmos** [comment]
@seladb  This PR is ready for merge.

File: README.md (hunk 2 above)
Comment (seladb) [inline README.md:21]:
```suggestion
Translations: English · [正體中文](./translation/README-zh-tw.md)
```

File: translation/README-zh-tw.md [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+
+[![X 關注](https://img.shields.io/badge/follow-%40seladb-1DA1F2?logo=x&style=social)](https://x.com/intent/follow?screen_name=seladb)
+[![GitHub Repo 星星](https://img.shields.io/github/stars/seladb/PcapPlusPlus?style=social)]()
+
+</div>
+
+[PcapPlusPlus](https://pcapplusplus.github.io/) 是一個跨平台的 C++ 函式庫，提供高效、強大且易於使用的功能，來進行網路封包的擷取、解析和生成。
+
+PcapPlusPlus 支援對多種網路協議進行解析和建構，並對常見的封包處理函式庫（如 [libpcap](https://www.tcpdump.org/)、[WinPcap](https://www.winpcap.org/)、[Npcap](https://nmap.org/npcap/)、[DPDK](https://www.dpdk.org/)、[eBPF AF_XDP](https://www.kernel.org/doc/html/next/networking/af_xdp.html) 和 [PF_RING](https://www.ntop.org/products/packet-capture/pf_ring/)）提供 C++ 的封裝函式。
+
```
Comment (seladb) [inline translation/README-zh-tw.md:19]: Add an option to move back to English:

```suggestion:
翻譯: [English](README.md) · 正體中文
```

File: README.md (hunk 2 above)
Comment (tigercosmos) [reply README.md]: Fixed.

File: translation/README-zh-tw.md (hunk 4 above)
Comment (tigercosmos) [reply translation/README-zh-tw.md:19]: Fixed.

**Review by seladb — APPROVED** [review]

**Pull request merged**